### DISCUSSION TITLE
POST request to the server

### DISCUSSION THREAD

**Discussion by Kolhun** [curl/curl#11595] [Q&A] [answered]
I'm trying to connect to my server, on other platforms the POST request worked properly, then it gives failed:Couldn't resolve host name

This is how it looks like my post request in the browser (JSON)
![image](https://github.com/curl/curl/assets/36815965/b51ae622-a6aa-46d2-aaee-61d8667927a4)

Here is the code from curl
![image](https://github.com/curl/curl/assets/36815965/dd513743-4bcb-4c34-ae2a-301cc14124e2)
![image](https://github.com/curl/curl/assets/36815965/cc92f1b0-df55-44c2-9f54-962e608c3fb1)

Question marks ("?????") for security purposes

**Comment by bagder** [comment, marked as answer]
The URL in your program is all messed up.

**Reply by Kolhun** [reply]
What does it mean?...

**Reply by Kolhun** [reply]
Sorry
